README.md

```diff
@@ -232,7 +232,9 @@ uploaded_image = image_client.upload_image(image)
 #### Annotate an image
 
 ```python
-from geti_sdk.data_models import Rectangle, Annotation
+from geti_sdk.data_models import Annotation
+from geti_sdk.data_models.shapes import Rectangle
```

Comment:
It's better to fix this import issue in `data_models/__init__.py`

Reply:
corrected